### DISCUSSION TITLE
How do I load rego files from an embedded file system for `rego.New`?

### DISCUSSION THREAD

**Discussion by darylhjd** [open-policy-agent/community#528] [OPA and Rego]
There seems to be a `loader.NewFileLoader().WithFS()` but I am not sure where this is supposed to go.

**Comment by anderseknert** [comment]
Yeah, something like `loader.NewFileLoader().WithFS(myFs).All([]string{"."})` to load everything in the embedded filesystem. Or you could read files one by one, and pass the string contents via `rego.Module`.

If you grep the OPA code and tests, you should find a few examples of the various loader functions and how they're used.
